pkg/apis/kubermatic/v1/policy_instance.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2025 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// PolicyInstanceResourceName represents "Resource" defined in Kubernetes.
+	PolicyInstanceResourceName = "policyinstances"
+
+	// PolicyInstanceKindName represents "Kind" defined in Kubernetes.
+	PolicyInstanceKindName = "PolicyInstance"
+)
+
+// Condition reasons for PolicyInstance.
+const (
+	// PolicyAppliedSuccessfully indicates the policy was successfully applied.
+	PolicyAppliedSuccessfully = "PolicyAppliedSuccessfully"
+
+	// PolicyApplicationFailed indicates the policy application failed.
+	PolicyApplicationFailed = "PolicyApplicationFailed"
+
+	// PolicyTemplateNotFound indicates the referenced template doesn't exist.
+	PolicyTemplateNotFound = "PolicyTemplateNotFound"
+)
+
+// Condition types for PolicyInstance.
+const (
+	// PolicyReady indicates if the policy has been successfully applied.
+	PolicyReadyCondition = "Ready"
+
+	// PolicyEnforced indicates if the policy is currently being enforced.
+	PolicyEnforcedCondition = "Enforced"
+)
+
+// +kubebuilder:resource:scope=Namespaced
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Enabled",type=boolean,JSONPath=".spec.enabled",description="Whether the policy is applied (only relevant if not enforced)"
+// +kubebuilder:printcolumn:name="Scope",type=string,JSONPath=".spec.scope",description="cluster or namespace"
+
+// PolicyInstance binds a PolicyTemplate to specific clusters/projects and
+// optionally enables or disables it (if the template is not enforced).
+type PolicyInstance struct {
```

Comment:
```suggestion
type PolicyBinding struct {
```

Let's have a better name